lib/ferver/file_list.rb

```diff
@@ -51,11 +41,11 @@ def all
     def find_files
       @files = []
 
-      Dir.foreach(configured_file_path) do |file|
-        next if file == '.' || file == '..'
+      Dir.foreach(configured_file_path) do |file_name|
+        next if file_name == '.' || file_name == '..'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.